### ISSUE TITLE
Ground the different types of groups / maturity stages in Problem Statements

### ISSUE THREAD

**Issue by michaelchampion** [w3c/w3process#813]
Inspired by https://kellanem.com/notes/new-tech and Related to https://github.com/w3c/w3process/issues/804 : It would clarify the Process and the "what type of group?" question to ground it in Problem Statements. 

Motivating problem: W3C's reputation for actually Leading the Web to Its Full Potential has suffered because there are not clear and rigorous criteria for what it means for various types of groups to exist and what it really means for a spec to be at a particular maturity level. Many CG specs or WG working drafts are de facto standards on the web, multiple groups appear to be working on the same problem with different approaches, and many Recommendations are aspirational statements of what could work rather than empirical statements of what actually works

- Creating a CG  should be fairly easy, it just requires a statement of a problem (or unrealized opportunity) with the current web platform / ecosystem and a suggestion to build a community to explore the problem and possible solutions. Participants agree to abide by the CoC/CEPC, work by some definition of "consensus", and to make RF patent commitment on their own contributions to the discussion. No explicit approval by the Team or other W3C bodies is needed for the CG to start working.

- Creating a WG is quite it bit harder. There should be a clear problem statement, evidence a community collectively addressing the problem, an incubated technical spec describing the broad outline of a solution, and a critical mass of platform implementers / website builders / users who plan to implement/deploy/use that solution once it is mature. Participants agree to work according to the W3C Process and make the commitments required by the Patent Policy. Approval of a WG charter requires consensus from the W3C membership that the WG would pursue a significant problem and there is a well-functioning community working together to solve it.
- Getting a Recommendation is quite difficult ... The WG must have created and tested a spec that is defined clearly enough to be interoperably implemented in all its details, there must be a real-world ecosystem of implementers and users that implement and use the spec, and there must be solid empirical evidence that implementations of the spec really improve the situation described in the CG->WG problem statement. Approval of a Recommendation requires strong and broad consensus from the W3C membership that the spec meets these criteria.

In short: CGs start with a problem and try to build a community to address it, WG's start with a community aligned around a rough/immature solution a problem, and Recommendations require strong evidence that the community really has addressed the problem they set out to solve.

**Comment by TallTed** [comment]
@michaelchampion — Your https://github.com/w3c/w3process/issues/813#issue-2057755326 has broken bullets. You need a space after each paragraph-starting hyphen. You also need to insert a blank line between the last bullet and the `In Short:`. These Markdown fixes will make that comment far easier to digest.

**Comment by frivoal** [comment]
Without bickering every word of that statement, in broad lines at least it seems sensible. But I am not sure what you mean to do by opening it as an issue. Do you think there's something in the process that contradicts it which we ought to fix? Something that's missing that we ought to add?

**Comment by michaelchampion** [comment]
I opened the issue to see If others think it is generally sensible to tweak the language of the Process document -- SHOULD statements, not MUST statements for the most part --to  encourage focusing on explicitly stated problems, and rationalizing advancement on the Rec Track in terms of a group's demonstrated ability to address the stated problem.

 As a general principle, I believe in getting rough consensus on issues before proposing specific language changes, but if there were rough consensus to proceed, the following sections would need updates:
 
 https://www.w3.org/2023/Process-20231103/#Intro - the "general overview of how W3C initiates standardization " have language along the lines of "CGs start with a problem and try to build a community to address it, WG's start with a community aligned around a rough/immature solution a problem, and Recommendations require strong evidence that the community really has addressed the problem they set out to solve".  A very rough first draft might be:
 
 > People generate interest in a particular problem (or unrealized opportunity) for users [possibly link to RFC 8890 or an authoritative W3C statement of the 'users first' principle ]of the currently deployed web platform. For instance, W3C Community members explore problems in  Community Groups [needs its own section in the Process] , or other public forums.   Also, the [Team](https://www.w3.org/2023/Process-20231103/#Team) may organize [Workshops](https://www.w3.org/2023/Process-20231103/#GAEvents) to bring people together to discuss problems that interest the W3C community.  If there is sufficient interest, the Team MAY propose an Interest Group to research the problem, understand "gaps" in the currently standardized web platform that inhibit solutions, and draft requirements for standards to fill the gaps.  Those interested in the problem SHOULD create one or more Community Groups to build a community and incubate at least rough drafts of specifications and implementations that would address it.

> When there is a well-functioning community addressing the problem and there is  a critical mass of users and implementers aligned around a possible W3C solution, [Team](https://www.w3.org/2023/Process-20231103/#team) works with the Membership to draft proposed [ Working Group charter](https://www.w3.org/2023/Process-20231103/#WGCharterDevelopment). W3C Members [review](https://www.w3.org/2023/Process-20231103/#CharterReview) the proposed charters, and when there is support within W3C for investing resources in the bringing the incubated solution to maturity, W3C approves the working group.

> When the working group has developed a specification to the Proposed Recommendation stage AND empirically demonstrated that the chartered work can meaningfully address the stated problem for actual web users, the WG and TEAM  initiate a request to [transition to Recommendation] ( https://www.w3.org/2023/Process-20231103/#transition-rec)

This implies other changes in the linked sections of the Process, especially:
- Creating a section to explicitly describe Community Groups and incorporate or reference the content of https://www.w3.org/community/about/process/ 

- The [ Working Group charter development](https://www.w3.org/2023/Process-20231103/#WGCharterDevelopment) section would need some editing to strongly encourage  community building around a problem and incubation of a solution before creating a WG, and incorporating an explicit problem statement into the charter.

- https://www.w3.org/2023/Process-20231103/#implementation-experience probably needs updating to include language indicating that there is real-world evidence that the implemented specification actually alleviates the problem noted  in the charter, for at least some identified set of real-world users. 

I suspect there are other parts of the Process that should be tweaked to "ground" it in solving problems for users rather than  implicit ["solutionism"]( https://en.wiktionary.org/wiki/solutionism)
